packages/assets/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "@datadog/build-plugins-assets",
+    "packageManager": "yarn@4.0.2",
+    "license": "MIT",
+    "private": true,
+    "author": "Datadog",
+    "exports": {
```

Comment:
FYI (not blocking): instead of using `exports` for such static files, you can use `files` to mark the `src/` folder as public
Or even don't write anything, and keep all the files top level in this package

Reply:
package.json should always be accessible
And if you want to remove those tsconfig and stuff, you can use either `files` in the package.json to opt in some, or use `.npmignore` to remove some on publish, see https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files